### PR TITLE
1.23.3 Release Notes

### PR DESCRIPTION
**Pull request by thedebugger**

### CODE REVIEW

**Comment by thedebugger** [comment]
Once this PR is ready to go, I'll cherrypick to release-1.23 and modify https://github.com/istio/istio.io/blob/release-1.23/data/args.yml#L5 with a make gen.

File: content/en/news/releases/1.23.x/announcing-1.23.3/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.23.2 and Istio 1.23.3
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** `clusterLocal` host exclusions for multi-cluster.
+
+- **Added** the metrics port in the daemonset containers spec of the istio-cni chart.
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.3/index.md:18]:
```suggestion
- **Added** the metrics port in the `DaemonSet` containers spec of the `istio-cni` chart.
```

File: content/en/news/releases/1.23.x/announcing-1.23.3/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- **Added** the metrics port in the kube-gateway containers spec of the istio-discovery chart.
+
+- **Fixed** kube-virt-interfaces rules not being removed by istio-clean-iptables tool.
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.3/index.md:22]:
```suggestion
- **Fixed** `kube-virt-interfaces` rules not being removed by `istio-clean-iptables` tool.
```

File: content/en/news/releases/1.23.x/announcing-1.23.3/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** `clusterLocal` host exclusions for multi-cluster.
+
+- **Added** the metrics port in the daemonset containers spec of the istio-cni chart.
+
+- **Added** the metrics port in the kube-gateway containers spec of the istio-discovery chart.
```
Comment (dhawton) [inline content/en/news/releases/1.23.x/announcing-1.23.3/index.md:20]:
```suggestion
- **Added** the metrics port in the `kube-gateway` container spec of the `istio-discovery` chart.
```

**Review by dhawton — APPROVED** [review]

**Pull request merged**